stemcell_builder/stages/bosh_go_agent/apply.sh

```diff
@@ -28,6 +28,11 @@ cd $assets_dir/go/src/github.com/cloudfoundry/bosh-agent
 bin/build
 
 mv out/bosh-agent $chroot/var/vcap/bosh/bin/
+if [ `uname -m` == "ppc64le" ]; then
+   # assume that gccgo is installed under /usr/local/gccgp
+   cp -rvH /usr/local/gccgo $chroot/var/vcap/bosh/
```

Comment:
@ipoddar-ibm could you please tell, why did you need copying gccgo folder to `/var/vcap/bosh/` ?